### ISSUE TITLE
Generate ToString for types

### ISSUE THREAD

**Issue by vasily-kirichenko** [fsprojects/SwaggerProvider#52]
Currently FSI shows results like this:

``` fsharp
val it : Service.xGetResponse [] =
  [|SwaggerProvider.Service+xGetResponse
      {Infos = [|SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo;
                     SwaggerProvider.Service+xGetFileInfo|];
       File = SwaggerProvider.Service+xGetFileFile;}|]
```

which makes using the TP in FSI meaningless. 

Suggestion: generate `ToString` overrides, similar to what we have for records, for all generated types.

**Comment by sergey-tihon** [comment]
What `ToString` should return? `operationId`?

**Comment by sergey-tihon** [comment]
Released in [v0.6.1](https://www.nuget.org/packages/SwaggerProvider/0.6.1)
